src/Nordea/Components/Common.elm

```diff
@@ -0,0 +1,161 @@
+module Nordea.Components.Common exposing (..)
+
+import Css
+    exposing
+        ( alignItems
+        , auto
+        , center
+        , color
+        , displayFlex
+        , flex
+        , flexBasis
+        , fontFamilies
+        , fontSize
+        , justifyContent
+        , lineHeight
+        , marginBottom
+        , marginLeft
+        , marginRight
+        , marginTop
+        , none
+        , pct
+        , rem
+        , spaceBetween
+        )
+import Html.Styled as Html exposing (Attribute, Html, div, styled, text)
+import Html.Styled.Attributes exposing (css)
+import Maybe.Extra as Maybe
+import Nordea.Html as Html exposing (showIf)
+import Nordea.Resources.Colors as Colors
+import Nordea.Resources.Icons as Icons
+
+
+type RequirednessHint
+    = Mandatory ({ no : String, se : String, dk : String } -> String)
+    | Optional ({ no : String, se : String, dk : String } -> String)
+    | Custom String
+
+
+type alias InputProperties =
+    { labelText : String
+    , requirednessHint : Maybe RequirednessHint
+    , showFocusOutline : Bool
+    , errorMessage : Maybe String
+    , hintText : Maybe String
+    , charCounter : Maybe CharCounter
+    }
+
+
+type alias CharCounter =
+    { current : Int
+    , max : Int
+    }
+
+
+{-| Should be replaced by common typography
+-}
+bodyText : List (Attribute msg) -> List (Html msg) -> Html msg
+bodyText =
+    styled Html.span
+        [ fontSize (rem 0.875)
+        , fontFamilies [ "Nordea Sans Small" ]
+        , lineHeight (rem 1.5)
+        , color Colors.black
+        ]
+
+
+bottomInfo : InputProperties -> List (Html msg)
+bottomInfo config =
+    let
+        charCounterView =
+            config.charCounter
+                |> Html.viewMaybe
+                    (\charCounter ->
+                        bodyText [ css [ marginLeft auto, color Colors.grayDark |> Css.important ] ]
+                            [ String.fromInt charCounter.current ++ "/" ++ String.fromInt charCounter.max |> text ]
+                    )
+    in
+    [ div
+        [ css
+            [ displayFlex
+            , flexBasis (pct 100)
+            , justifyContent spaceBetween
+            , marginTop (rem 0.5)
+            , Css.property "gap" "1rem"
```

Comment:
Can `(rem 1)` be used insted of `"1rem"`?

Reply:
I do not think so. As I understood it, the "gap" property is only available through this Css.property and it takes two strings as parameters and need to be written in same way as in CSS.

Reply:
👍